src/style.css

```diff
@@ -1,3 +1,88 @@
+html {
+  /* stylelint-disable-next-line */
+  font-family:
+  Roboto, Arial,
+  Helvetica Neue,
+  Helvetica,
+  sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+
+  --active-link-color: #00acdc;
+  --header-height: 60px;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+}
+
+.logo {
+  display: inline-block;
+  line-height: 0;
+}
+
+.figure__logo-moyo {
+  margin: 0;
+}
+
+.nav__list {
```

Comment:
Don't use tag names for styling (except `html` and `body`). It's better to add a class to the `ul` element and style that class instead of using the tag name directly.